package.json

```diff
@@ -64,8 +64,7 @@
   "scripts": {
     "postinstall": "./webpack_if_prod.sh",
     "lint": "node ./node_modules/eslint/bin/eslint.js ./static/js",
-    "test": "node ./node_modules/mocha/bin/_mocha --compilers js:babel-register --require static/js/global_init.js 'static/**/*/*_test.js'",
-    "singleTest": "node ./node_modules/mocha/bin/_mocha --compilers js:babel-register --require static/js/global_init.js $1",
+    "test": "./js_test.sh",
```

Comment:
can you not use this nifty syntax `${1:-'static/**/*/*_test.js'}` directly in the command definition in package.json? it's not so bad to have these sh files around but i feel like we should avoid it if possible